File: Act-1-POO/Act-1-POO-Phyton/Director.py
class Director:
	def __init__(self, id_director, nombre_director, edad_director, premios_director, pelicula_dirigida):
		self.id_director = id_director
		self.nombre_director = nombre_director
		self.edad_director = edad_director
		self.premios_director = premios_director
		self.pelicula_dirigida = pelicula_dirigida

director = Director(
	id_director=3000000,
	nombre_director="Francis Ford Coppola",
	edad_director="85",
	premios_director="5 Oscar",
	pelicula_dirigida="El padrino"
)

print(director.nombre_director)

File: Act-1-POO/Act-1-POO-PHP/Director.php
<?php

class Director {
	private $IdDirector;
	private $NombreDirector;
	private $EdadDirector;
	private $PremiosDirector;
	private $PeliculaDirigida;

	// Constructor
	public function __construct($id, $nombre, $edad, $premios, $pelicula) {
		$this->IdDirector = $id;
		$this->NombreDirector = $nombre;
		$this->EdadDirector = $edad;
		$this->PremiosDirector = $premios;
		$this->PeliculaDirigida = $pelicula;
	}

	// Getters
	public function getIdDirector() {
		return $this->IdDirector;
	}

	public function getNombreDirector() {
		return $this->NombreDirector;
	}

	public function getEdadDirector() {
		return $this->EdadDirector;
	}

	public function getPremiosDirector() {
		return $this->PremiosDirector;
	}

	public function getPeliculaDirigida() {
		return $this->PeliculaDirigida;
	}

	// Setters
	public function setIdDirector($id) {
		$this->IdDirector = $id;
	}

	public function setNombreDirector($nombre) {
		$this->NombreDirector = $nombre;
	}

	public function setEdadDirector($edad) {
		$this->EdadDirector = $edad;
	}

	public function setPremiosDirector($premios) {
		$this->PremiosDirector = $premios;
	}

	public function setPeliculaDirigida($pelicula) {
		$this->PeliculaDirigida = $pelicula;
	}
}

?>
